### PR TITLE
Add problems from years 2015-2019 & Fix typo at readme

### PR DESCRIPTION
**Pull request by Alves-Bruno**

### CODE REVIEW

File: 2015/serial/A/Makefile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+.PHONY = all clean
+
+all : program
+
+program : huffman.cc
+	icpc -std=c++11 -Wall -O3 huffman.cc -o huffman
```
Comment (648trindade) [inline 2015/serial/A/Makefile:6]: Please change all Makefile `icc` and `icpc` references to `gcc` and `g++` references, respectively.
Also make sure that all problems can be built and run with the inputs provided.

File: 2015/misc/inputs/C.in [hunk 2]
```diff
@@ -0,0 +1,4 @@
+3
+1 3 5
+2 4 7
+1 1 0
```
Comment (648trindade) [inline 2015/misc/inputs/C.in:4]: Please remove all these duplicated input files. All input files should be placed at `misc/inputs` directory

File: 2019/serial/D/Makefile [hunk 3]
```diff
@@ -0,0 +1,15 @@
+# Makefile for simulation
+
+LDFLAGS=-lm
+CFLAGS = -g -Wall -Wno-deprecated -O3 -march=native -std=c11
```
Comment (648trindade) [inline 2019/serial/D/Makefile:4]: Make sure that no serial problem is being built with `-g` or `-O0` flags. Replace them with `-O2` flags

**Review by 648trindade — CHANGES_REQUESTED** [review]
Thank @Alves-Bruno ! That's a huge contribution!

Please add a `X.sh` (where `X` is the problem letter) shell script at each problem, in order that we can make it run by simply calling
```
bash X.sh
```
without specifying the input file or any other runtime environment variables (all these stuff should be done by the script)

You can use the shell scripts from 2006 and 2008 problems as a base

**Comment by Alves-Bruno** [comment]
This PR was too big. I'll do new ones with smaller contributions. Thanks @648trindade  for the advice.

**Pull request closed without merging**